### ISSUE TITLE
HHVM - pgsql extension

### ISSUE THREAD

**Issue by zoltan-schwarz** [titon/nexus#2]
The box is extremely usefull and it contains all what I needed. Soo there is a great thanks to you for it. But yesterday when I tried to connect to the PostgreSql database I found, there is no hhvm pgsql extension installed.

**Comment by milesj** [comment]
By default, HHVM itself does not support PostgreSQL. The only pgsql support there is is this extension: https://github.com/PocketRent/hhvm-pgsql 

Which is unofficial, does not support all functionality, and usually breaks every new HHVM release. Until an official port is provided by HHVM, I feel like supporting this extension should be on the hands of the developer.

**Comment by zoltan-schwarz** [comment]
Yeah, I know its unofficial extension. But we using it in live environment for 4-5 years. This is why I thought. But you are right.

**Comment by milesj** [comment]
I'll leave this open for now and if it gets a bit more support, I'll look into integrating it by default.

**Comment by zoltan-schwarz** [comment]
I'll be the happiest :) Because I don't have much experience setting up webservers :) But as I see, most of the box users didnt know about this repo on github.
